demo/dotnet/CobraDemo/FileDemo.cs

```diff
@@ -199,6 +209,7 @@ private static void OnUnhandledException(object sender, UnhandledExceptionEventA
         private static readonly string HELP_STR = "Available options: \n " +
             $"\t--input_audio_path (required): Absolute path to input audio file.\n" +
             $"\t--access_key (required): AccessKey obtained from Picovoice Console (https://console.picovoice.ai/)\n" +
+            $"\t--device: device to run demo\n" +
```

Comment:
Helper text is not very helpful. Copy the helper text from completed products.

You also need to add `show_inference_devices`